### DISCUSSION TITLE
Data references for NAV

### DISCUSSION THREAD

**Discussion by hudkinsnoah** [JSBSim-Team/jsbsim#921] [Q&A]
I am working on simulating NAV data which requires specific information which, when using XPlane, just comes from the built in NAV radio. Is there anything similar to this built in to JSBSim? Or does this data need to come from elsewhere?

Thank you!

**Comment by seanmcleod** [comment]
This sort of thing isn't built into JSBSim, a developer would need to add it themselves, possibly using the systems facility of JSBSim. If you don't want to do it from scratch then take a look at some of the aircraft, systems etc. in FlightGear. A large number of the FlightGear aircraft make use of a JSBSim FDM but will have more complete systems in terms of navigation systems etc. and possibly some data supplied by FlightGear.
